src/Providers/Stripe/Token.php

```diff
@@ -2,15 +2,18 @@
 
 namespace PodPoint\Payments\Providers\Stripe;
 
-use PodPoint\Payments\Providers\Stripe\Token as StripeToken;
-
 class Token extends \PodPoint\Payments\Token
 {
     const CUSTOMER = 'customer';
+    const TOKEN = 'token';
     const PAYMENT_INTENT = 'payment_intent_id';
+    const SECRET_PAYMENT_INTENT = 'secret_payment_intent';
     const PAYMENT_METHOD = 'payment_method';
-    const CHARGE = 'charge';
+    const SETUP_INTENT = 'setup_intent_id';
+    const SECRET_SETUP_INTENT = 'secret_setup_intent';
     const CARD = 'card';
+    const CHARGE = 'charge';
+    const UNDEFINED = 'undefined';
```

Comment:
I think this should be changed if we are only using it for set up?

Reply:
not a fan of this either

Reply:
I'm going to rename it to SETUP_CARD_CREATION.